### ISSUE TITLE
Add .lha file type to Amiga file types

### ISSUE THREAD

**Issue by retrogamecorps** [eduardofilo/RG350_adam_image#16]
Hi, .lha files work well with RetroArch (although many Amiga games run slowly on this chipset).  Would you mind adding that file extension to the .simplemenu/section_groups/home computers.ini file?  Thanks :)

**Comment by eduardofilo** [comment]
Hi.
Amiga extensions are certainly wrong. In the table I say that these extensions are supported:
```
adf, adz, dms, fdi, ipf, hdf, hdz, lha, slave, info, cue, ccd, nrg, mds, iso, chd, uae, m3u, zip, 7z
```
but then I only included:
```
.adf,.zip,.cue,.chd,.iso,.adz,.dms,.fdi,.ipf,.hdf
```

I will fix this in the next release. Thanks...

**Comment by eduardofilo** [comment]
Fixed in v1.2
